monitoring/api/AlertSample/Program.cs

```diff
@@ -359,5 +396,29 @@ static object EnablePolicies(string projectId, string filter, bool enable)
         }
         // [END monitoring_alert_enable_policies]
         // [END monitoring_alert_disable_policies]
+
+        static void DeleteAlertPolicy(string policyNameString)
+        {
+            var client = AlertPolicyServiceClient.Create();
+            AlertPolicyName policyName;
+            if (!AlertPolicyName.TryParse(policyNameString, out policyName))
+            {
+                throw new BadNameException();
+            }
+            client.DeleteAlertPolicy(policyName);
+            Console.WriteLine($"Deleted {policyName}.");
+        }
+        static void DeleteNotificationChannel(string channelNameString, bool force)
+        {
+            var client = NotificationChannelServiceClient.Create();
+            NotificationChannelName channelName;
+            if (!NotificationChannelName.TryParse(channelNameString, out channelName))
+            {
+                throw new BadNameException();
+            }
+            client.DeleteNotificationChannel(channelName, force);
+            Console.WriteLine($"Deleted {channelName}.");
+        }
+        public class BadNameException : Exception { }

```

Comment:
since it's a sample code, maybe worth throwing standard exception with the right message to not complicate things?

Reply:
Done.